### ISSUE TITLE
H

### ISSUE THREAD

**Issue by karloxloma** [HeliosVirtualCockpit/Helios#853]
I am reaching out to inquire whether your software is capable of running on two computers connected via a network. Specifically, I would like to understand if the software supports sending the data to localhost and an external pc on the same LAN.

I tried modifying:

helios_private.host = "192.168.0.107" to:

helios_private.host = "192.168.0.107,127.0.0.1"

Also with semicolon and with two independent lines, none of them works

Thanks

**Comment by BlueFinBima** [comment]
@karloxloma No, HeliosExport16.lua only supports sending to a single endpoint.  Probably the simplest approach is to duplicate  HeliosExport16.lua (and hard code the second IP address into that file).  You might then add the duplicated version of HeliosExports16.lua into your Exports.lua.  I have not tested this, and cannot be certain that this will work.  
Helios is a community effort and discussions and questions such as this are best directed at the Discord server [Wiki Section about getting help](https://github.com/HeliosVirtualCockpit/Helios/wiki/Get-Help#7-if-you-need-help-because-you-cant-figure-out-how-to-do-something-in-helios)
